### PR TITLE
Fix env var usage

### PR DESCRIPTION
**Pull request by kripnerl**
# Changes: 

- [Use safer os.environ.get method and handle undefined variables where needed](https://github.com/gafusion/omas/commit/722db18ba5f8787abd2decb66df9742038af2b40)
- use `pathlib.Path.home()` inseted of `os.environ["HOME"]`  to be compatible with Windows. 
- [Removed not existing variable.](https://github.com/gafusion/omas/commit/2d8a8e4f88e7e9c5f3541657d7111c420e0d5af6)

# Note: 

Nevertheless, Windows compatibility was not tested since I do not have access to Windows at the moment.

### CODE REVIEW

File: omas/tests/failed_imports.py [hunk 1]
```diff
@@ -40,7 +42,7 @@
     from botocore.exceptions import NoCredentialsError
     import boto3
 
-    if not os.path.exists(os.environ.get('AWS_CONFIG_FILE', os.environ['HOME'] + '/.aws/config')):
+    if not os.path.exists(os.environ.get('AWS_CONFIG_FILE', pathlib.Path.home() / '/.aws/config')):
```
Comment (orso82) [inline omas/tests/failed_imports.py:45]:
```suggestion
    if not os.path.exists(os.environ.get('AWS_CONFIG_FILE', pathlib.Path.home() + '/.aws/config')):
```

**Review by orso82 — COMMENTED** [review]
looks great! One small typo.

**Comment by orso82** [comment]
Regressions test fail. Sorry, I have never used `pathlib` and did not realize it does not behave like a standard string :/

```
ImportError: Failed to import test module: failed_imports
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.6.15/x64/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "/opt/hostedtoolcache/Python/3.6.15/x64/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/home/runner/work/omas/omas/omas/tests/failed_imports.py", line 45, in <module>
    if not os.path.exists(os.environ.get('AWS_CONFIG_FILE', pathlib.Path.home() + '/.aws/config')):
TypeError: unsupported operand type(s) for +: 'PosixPath' and 'str'

```

File: omas/tests/failed_imports.py (hunk 1 above)
Comment (kripnerl) [reply omas/tests/failed_imports.py]: Uhh it was not typo. It was correct. I have rewrote both parts which are using `pathlib`

**Comment by kripnerl** [comment]
> Regressions test fail. Sorry, I have never used `pathlib` and did not realize it does not behave like a standard string :/
> 
> ```
> ImportError: Failed to import test module: failed_imports
> Traceback (most recent call last):
>   File "/opt/hostedtoolcache/Python/3.6.15/x64/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
>     module = self._get_module_from_name(name)
>   File "/opt/hostedtoolcache/Python/3.6.15/x64/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
>     __import__(name)
>   File "/home/runner/work/omas/omas/omas/tests/failed_imports.py", line 45, in <module>
>     if not os.path.exists(os.environ.get('AWS_CONFIG_FILE', pathlib.Path.home() + '/.aws/config')):
> TypeError: unsupported operand type(s) for +: 'PosixPath' and 'str'
> ```

My bad - I was a bit confused by the suggestion :-). Hopefully, it's fixed now.

**Comment by kripnerl** [comment]
Btw `pathlib` is a very handy library - makes life much simpler!

**Pull request merged**